### PR TITLE
expose raft's observer API via store

### PR DESCRIPTION
**Pull request by tych0**
Signed-off-by: Tycho Andersen <tycho.andersen@canonical.com>

### CODE REVIEW

File: store/store.go [hunk 1]
```diff
@@ -769,6 +769,10 @@ func (s *Store) Restore(rc io.ReadCloser) error {
 	}()
 }
 
+func (s *Store) RegisterObserver(o *raft.Observer) {
```
Comment (otoolep) [inline store/store.go]: Can you add a GoDoc string? E.g. `RegisterObserver registers an observer that ....`

**Comment by otoolep** [comment]
I'll tweak it myself.

**Comment by tych0** [comment]
Hey!

Apologies for the delay, I was traveling. Thanks for the merge though!

Tycho

**Comment by otoolep** [comment]
No worries. Small tweak.

**Pull request merged**